atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java

```diff
@@ -189,7 +189,15 @@ private static void login(Client client, CassandraCredentialsConfig config) thro
 
     @Override
     public boolean validateObject(PooledObject<CassandraClient> client) {
-        return client.getObject().getOutputProtocol().getTransport().isOpen();
+        try {
+            return client.getObject().getOutputProtocol().getTransport().isOpen();
+        } catch (Throwable t) {
+            log.info(
```

Comment:
Should we warn? Is there any safe information we can log about the client?

Reply:
I think info is OK, as it doesn't indicate anything bad about this specific user node / that the health is trending badly.

Client information: yeah, we can add the host (as we do in `destroyObject()`) - I've added this.